cmd/internal/dgather.go

```diff
@@ -2,17 +2,18 @@ package cmd
 
 import (
 	"bufio"
+	"bytes"
 	"encoding/json"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
 
+	"code.cloudfoundry.org/cf-operator/pkg/bosh/manifest"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	yaml "gopkg.in/yaml.v2"
-
-	"code.cloudfoundry.org/cf-operator/pkg/bosh/manifest"
```

Comment:
While you're at it, can you move that back to line 15? 
It's only style, but we group import statements like go, github, k8s, our code in a lot of files.

Reply:
done